docs/modules/ROOT/pages/adr/0034-vshnredis-high-availability.adoc

```diff
@@ -0,0 +1,83 @@
+= ADR 0034 - Redis High Availability /  Clustering
+:adr_author:    Nicolas Bigler
+:adr_owner:     Schedar
+:adr_reviewers: 
+:adr_date:      2025-06-04
+:adr_upd_date:  2025-06-04
+:adr_status:    draft
+:adr_tags:      redis,ha
+
+include::partial$adr-meta.adoc[]
+
+[NOTE]
+.Summary
+====
+This ADR evaluates different Redis HA deployment options on Kubernetes to support a production-grade, fault-tolerant Redis setup. The goal is to ensure high availability, resilience to node failures, and ease of management/operations.
+====
+
+== Context
+
+Requirements to the Redis HA architecture:
+
+* Minimizes downtime during failover
+* Supports Kubernetes-native patterns
+* Is operationally manageable
+* Is compatible with our existing infrastructure and client usage patterns
+
+=== Considered Options
+
+**Option 1:  Redis Sentinel + Redis Master/Replica**
```

Comment:
```suggestion
Option 1:  Redis Sentinel + Redis Master/Replica::
```

Use this syntax throughout the document, ref: https://docs.asciidoctor.org/asciidoc/latest/lists/description/